apps/api/src/campaign/dto/create-campaign.dto.ts

```diff
@@ -98,6 +98,13 @@ export class CreateCampaignDto {
   @Type(() => Date)
   endDate: Date | null
 
+  @ApiProperty()
+  @Expose()
+  @IsOptional()
+  @IsString()
+  @IsEnum(CampaignState)
+  state: CampaignState | undefined
```

Comment:
When we add the control of the state of a campaign to the frontend we open the gates for intervention and for example submit a campaign as `approved`.

@quantum-grit Have you considered this when you've updated the code?